my_list.rb

```diff
@@ -0,0 +1,25 @@
+class MyList
+  require './my_enumerable'
```

Comment:
- [ ] Let's ensure that you use `require_relative` to import `my_enumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention. "`require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory." See suggestion:

```suggestion
  require_relative 'my_enumerable'
```

Reply:
Oh, I see, thanks for your suggestion @julie-ify 😄